README.md

```diff
@@ -131,12 +131,12 @@ The command line tool can also:
   $ kni -J hello.json
   ```
 
-- Kni can also produce a diagnostique view of a story. The first column is
```

Comment:
maybe this one was intended; let me know if

Reply:
I can no longer tell the difference between “intentional but incorrect” from “I don’t know how to spell properly” anymore.